tests/test_order_lines.py

```diff
@@ -0,0 +1,62 @@
+from mollie.api.objects.order_line import OrderLine
+
+from .utils import assert_list_object
+
+ORDER_ID = 'ord_pbjz8x'
+LINE_ID = 'odl_dgtxyl'
+
+
+def test_get_order_lines(client, response):
+    """Retrieve lines of a single order by order ID."""
+    response.get('https://api.mollie.com/v2/orders/{order_id}'.format(order_id=ORDER_ID), 'order_single')
+
+    order = client.orders.get(ORDER_ID)
+    lines = order.lines
+    assert_list_object(lines, OrderLine)
+
+    # Test properties of the first line
+    line = next(lines)
+    assert isinstance(line, OrderLine)
+    assert line.id == LINE_ID
+    assert line.resource == 'orderline'
+    assert line.order_id == ORDER_ID
+    assert line.type == 'physical'
+    assert line.name == 'LEGO 42083 Bugatti Chiron'
+    assert line.status == 'created'
+    assert line.is_cancelable is True
+    assert line.quantity == 2
+    assert line.quantity_shipped == 0
+    assert line.amount_shipped == {'value': '0.00', 'currency': 'EUR'}
+    assert line.quantity_refunded == 0
+    assert line.amount_refunded == {'value': '0.00', 'currency': 'EUR'}
+    assert line.quantity_canceled == 0
+    assert line.amount_canceled == {'value': '0.00', 'currency': 'EUR'}
```

Comment:
Just a heads-up: after adding the new order line properties (`shippableQuantity`, `refundableQuantity` and `cancelableQuantity`) you should add some assertions here.